index.js

```diff
@@ -3073,38 +3087,78 @@
   //.
   //. > S.last ([])
   //. Nothing
+  //.
+  //. > S.last (Cons ('baz') (Cons ('foo') (Nil)))
+  //. S.Just ('foo')
```

Comment:
I'd prefer to see `S.last (Cons ('foo') (Cons ('bar') (Cons ('baz') (Nil))))` here. Deviating from the customary _foo_, _bar_, _baz_ order could lead to confusion.

Reply:
The expected output should now be `Just ('baz')`, I believe. You could run `npm run doctest` to check.

Reply:
🆗 ok